packages/run-protocol/src/vaultFactory/interest.js

```diff
@@ -15,6 +15,12 @@ const makeResult = (latestInterestUpdate, interest, newDebt) => ({
   newDebt,
 });
 
+// What about leap years?
+// https://www.grc.nasa.gov/www/k-12/Numbers/Math/Mathematical_Thinking/calendar_calculations.htm
+// says the average Gregorian year is 365.2425
+// whereas the astronomical figure (tropical year) is 365.2422
+// No mention made of the occasional leap second!
+// Whether we care depends on what we do with it.
```

Comment:
I'll admit this is overly pedantic and probably doesn't matter at all. But I am curious if, for our use, there is an unambiguous right answer.

Reply:
I had wondered the same thing. It's a negligible difference and for loans the regulations allow treating interest in a leap year as any other.
https://www.consumerfinance.gov/rules-policy/regulations/1030/interp-7/#:~:text=Leap%20year.,earn%20interest%20for%20February%2029.
> 4. Leap year. Institutions may apply a daily rate of 1/366 or 1/365 of the interest rate for 366 days in a leap year, if the account will earn interest for February 29.